packages/server/tests/acceptance/conformityTests.spec.ts

```diff
@@ -242,11 +244,11 @@ function checkResponseFormat(actualReponse, expectedResponse) {
   }
 }
 
-function findSchema(file) {
+const findSchema = function (file) {
```

Comment:
is there a specific reason we changed thi?

Reply:
I was testing different approaches. with the one uploaded we don't need it, so I'll revert the change. thanks!